examples/st7789_172x320_1.47_simpletest.py

```diff
@@ -3,25 +3,36 @@
 
 """
 This test will initialize the display using displayio and draw a solid green
-background, a smaller purple rectangle, and some yellow text.
+background, a smaller purple rectangle, and some yellow text..
 """
 import board
+import busio
 import terminalio
 import displayio
 from adafruit_display_text import label
 from adafruit_st7789 import ST7789
 
-BORDER_WIDTH = 20
+BORDER_WIDTH = 28
 TEXT_SCALE = 3
 
 # Release any resources currently in use for the displays
 displayio.release_displays()
 
-spi = board.SPI()
+# built-in, silkscreen labelled SPI bus  
+spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
```

Comment:
Please use `board.SPI()` it is the standard default everywhere.
```suggestion
spi = board.SPI()
```
By using the board singleton, it makes it easier to share code between different projects.

Reply:
This is changed back to board.SPI() with the latest commits.